src/ansys/mechanical/core/embedding/app.py

```diff
@@ -201,6 +201,8 @@ def __init__(self, db_file=None, private_appdata=False, **kwargs):
                 if globals:
                     # The next line is covered by test_globals_kwarg_building_gallery
                     instance.update_globals(globals)  # pragma: nocover
+                # Set messages to None
+                self._messages = None
```

Comment:
I prefer to put `self._messages = None` one time before the check for if `BUILDING_GALLERY`